Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -58,7 +58,7 @@ ref class AppEventListener
 
 extern "C" void _ApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg);
```

Comment:
No, this is the entry for functional tests

---
In reply to: [72336476](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72336476) [](ancestors = 72336476)